src/components/templates/OrderTemplate.jsx

```diff
@@ -0,0 +1,201 @@
+import { useMutation } from "react-query";
+import { comma } from "../../utils/convert";
+import { order } from "../../services/order";
+import { useNavigate } from "react-router-dom";
+import { useRef, useState } from "react";
+import InputGroup from "../molecules/InputGroup";
+
+const OrderTemplate = ({ data }) => {
+  const { products, totalPrice } = data?.data?.response;
+  const navigate = useNavigate();
+  const [agreePayment, setAgreePayment] = useState(false);
+  const [agreePolicy, setAgreePolicy] = useState(false);
+
+  const allAgreeRef = useRef(null);
+  const agreePaymentRef = useRef(null);
+  const agreePolicyRef = useRef(null);
+
+  // All-agree
+  const handleAllAgree = (e) => {
+    const value = e.target.checked;
+    setAgreePayment(value);
+    setAgreePolicy(value);
+  };
+
+  // part-agree
+  const handleAgreement = (e) => {
+    const { name, checked } = e.target;
+
+    if (name === "payment-agree") {
+      setAgreePayment(checked);
+    } else if (name === "policy-agree") {
+      setAgreePolicy(checked);
+    }
+  };
+
+  const { mutate } = useMutation({
+    mutationKey: "order",
+    mutationFn: order,
+  });
+
+  //목표
+  //상품명을 적절하게 표기
+  // 그에 따라, 가격, 수량
+
+  // products  안 에 있는 item
+  // `${item.productName} - ${item.carts[0].option.optionName}`
+  // 1개당 가격 : item.carts[0].price * item.carts[0].quantity
+
+  const OrderItems = () => {
+    let renderComponent = [];
+    if (Array.isArray(products) === false) return;
```

Comment:
```suggestion
    if (!Array.isArray(products)) return;
```
반환 값이 불리언이기때문에 false를 굳이 명시해주실 필요는 없겠습니다.